graceful_controller_ros/src/graceful_controller_ros.cpp

```diff
@@ -355,8 +355,17 @@ geometry_msgs::msg::TwistStamped GracefulControllerROS::computeVelocityCommands(
   // Compute distance to goal
   double dist_to_goal = std::hypot(goal_pose.pose.position.x, goal_pose.pose.position.y);
 
+  // Compute distance to stopped
+  bool below_velocity_limits = true;
+  if (fabs(velocity.linear.x) > velocity_tolerance.linear.x ||
+      fabs(velocity.angular.z) > velocity_tolerance.angular.z)
```

Comment:
I'm now thinking this should be implemented differently - there really isn't a good reason we should use the overall goal velocity - these should probably be separate parameters.